data/pools/transactionPool.go

```diff
@@ -435,6 +301,92 @@ func (pool *TransactionPool) OnNewBlock(block bookkeeping.Block) {
 	}
 }
 
+// alwaysVerifiedPool implements ledger.VerifiedTxnCache and returns every
+// transaction as verified.
+type alwaysVerifiedPool struct{}
+
+func (*alwaysVerifiedPool) Verified(txn transactions.SignedTxn) bool {
+	return true
+}
+
+func (pool *TransactionPool) addToPendingBlockEvaluatorOnce(tx *transactions.SignedTxn) error {
```

Comment:
Curious - what does 'once' mean here?

Reply:
There's an `addToPendingBlockEvaluator` that retries adding to the block evaluator if it's out of space in the current "simulated" block, and it calls this `...Once()` function again.